tofu/imas2tofu/_core.py

```diff
@@ -1487,19 +1487,22 @@ def get_events(self, occ=None, verb=True, returnas=False):
         names, times = names[ind], times[ind]
 
         # print and / or return as list / tuple
-        if verb:
+        if verb is True or returnas is str:
             msg = np.array([range(times.size), names, times], dtype='U').T
             length = np.nanmax(np.char.str_len(msg))
             msg = np.char.ljust(msg, length)
             msg = ('index'.ljust(length) + ' name'.ljust(length)
                    + '  time ({})'.format(tunits).ljust(length)
                    + '\n' + ' '.join(['-'*length for ii in [0, 1, 2]]) + '\n'
                    + '\n'.join([' '.join(aa) for aa in msg]))
+        if verb is True:
```

Comment:
`if verb` ?

Reply:
verb is a keyword arg used to activate / deactivate verbosity (i.e.: message printing)

Reply:
Sorry my comment was not clear enough. I meant could you write only `if verb` instead of `if verb is True`

Reply:
Yes an no...
if the user passes, by mistake, something than a bool ( a int or a float for example) if verb will be true (except if the float is 0).
It depends on the behaviour we want, shall we ask the user to pass a bool or shall anything else work too in case he / she makes a mistake ? 
The first case `if verb is True` is cleaner but maybe a bit strict
The second case `if verb `is more flexible but it may happen that the user doesn't understand what he / she is doing

Honestly I'm not sure which one is the best